orekit/tests/validate_twobody/validate_propagation.py

```diff
@@ -0,0 +1,55 @@
+""" Validate poliastro's two-body propagation against Orekit"""
+
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+from org.hipparchus.geometry.euclidean.threed import Vector3D
+from org.orekit.frames import FramesFactory
+from org.orekit.orbits import KeplerianOrbit
+from org.orekit.propagation.analytical import KeplerianPropagator
+from org.orekit.time import AbsoluteDate
+from org.orekit.utils import Constants as C
+from org.orekit.utils import PVCoordinates
+from poliastro.bodies import Earth
+from poliastro.twobody import Orbit
+
+import orekit
+from orekit.pyhelpers import setup_orekit_curdir
+
+# Setup orekit virtual machine and associated data
+VM = orekit.initVM()
+setup_orekit_curdir("orekit-data.zip")
+
+
+def validate_elliptic_propagation():
+    # Initial orbit state vectors, final radius and time of flight
+    rx_0, ry_0, rz_0 = [float(ri_0) for ri_0 in [1131.340e3, -2282.343e3, 6672.423e3]]
+    vx_0, vy_0, vz_0 = [float(vi_0) for vi_0 in [-5.64305e3, 4.30333e3, 2.42879e3]]
```

Comment:
Why do we need this list comprehension instead of doing

```suggestion
    rx_0, ry_0, rz_0 = 1131.340e3, -2282.343e3, 6672.423e3
    vx_0, vy_0, vz_0 = -5.64305e3, 4.30333e3, 2.42879e3
```

?

Reply:
For some reason, the Hipparchus math library from Java (the one Orekit makes use of) complains about if numbers being passed are not `float` kind. Sometimes, even if they are, I was required to use the float()` function just to make the code pass successfully...  Anyway, in this particular lines this behavior does not apply.

In the `validate_elements.py`, where initial position vectors components are `int` instances, I just added a floating point and tests were finally able to pass.